spec/fixtures/backup_fixtures/.tmp/backups/.gitkeep

```diff
@@ -1 +0,0 @@
-
```

Comment:
The specs delete this file some times, and when i re-create it i sometimes use echo '' > .gitkeep instead of just touching the file :stuck_out_tongue: